releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -8,7 +8,7 @@
 * The APIs `AzureAppConfigurationRefreshOptions.SetCacheExpiration` and `FeatureFlagOptions.CacheExpirationInterval` have been deprecated and will be removed in a future release. They are replaced with `AzureAppConfigurationRefreshOptions.SetRefreshInterval` and `FeatureFlagOptions.SetRefreshInterval`, respectively. This change does not affect functionality but aims to eliminate confusion regarding cache expiration implied by the previous API names. [#350](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/350)
 
 ### Enhancements
-* **Variant Feature Flags**: This is the first stable release with support for the `variants`, `allocation`, and `telemetry` properties for feature flags that were introduced in 7.1.0-preview. This enables [experimentation with Azure App Configuration](https://learn.microsoft.com/en-us/azure/azure-app-configuration/concept-experimentation), which opens up possibilities for improving user experience or software functionality using data from your feature flags.
+* **Variant Feature Flags**: This is the first stable release with support for the `variants`, `allocation`, and `telemetry` properties for feature flags added by the [Microsoft.FeatureManagement](https://github.com/microsoft/FeatureManagement-Dotnet) library. This enables [experimentation with Azure App Configuration](https://learn.microsoft.com/en-us/azure/azure-app-configuration/concept-experimentation), which opens up possibilities for improving user experience or software functionality using data from your feature flags.
```

Comment:
Yea you're right, I wanted to mention version 4.0.0 specifically and link it, but we have to wait for that release. I think I prefer that over saying something like "requires version 4"

Reply:
Any reason we don't want to make it clear that the v4 FM is required?

Reply:
I don't mind the idea, I just wasn't sure how to word it. 8.0.0 is compatible with 4.0.0-preview3 and newer, not all of v4 preview because of the new flag schema.

Reply:
We are talking about the stable release, so yes, we will only reference the 4.0.0 stable. Once a stable is released, we will not point anyone to any preview releases.